tools/perspective-scripts/lint_cpp.mjs

```diff
@@ -62,24 +62,17 @@ function tidy(buildDir, sourceDir, flags) {
     }
 }
 
-const CLANG_TIDY_PATH = fs.existsSync(`${__dirname}/../../.llvm`)
```

Comment:
My observation was that `run-clang-tidy` failed unless `clang-tidy` was also on the PATH. So I had to add `.llvm` to the PATH anyway to get it to build, making this logic redundant